src/main/java/com/star_trello/darkside/repo/QueueRepo.java

```diff
@@ -23,4 +23,11 @@
         return getByTitle(title).getTaskList();
     }
 
+    default Queue getQueueInfo(String title) {
+        Queue queue = getByTitle(title);
+        queue.setCreator(null);
```

Comment:
why we need set this to null? It is by default, isn't it?

Reply:
no, that handler returns creator credentials

Reply:
but, maybe in future i'll delete this handler, i'm not sure, will it need to me